www/js/splash/startprefs.ts

```diff
@@ -0,0 +1,121 @@
+import { storageGet, storageSet } from '../plugin/storage';
+import { logInfo, logDebug, displayErrorMsg } from '../plugin/logger';
+
+type StartPrefs = {
+  CONSENTED_EVENT: string,
+  INTRO_DONE_EVENT: string,
+}
```

Comment:
I don't think this type is necessary. Since `startPrefs` is declared outright right after this, Typescript already knows it's an obj with these 2 props and that the values of the props are `string`.

If you ever need to use that type signature for something else, you can do `typeof startPrefs`

Reply:
took that out in [3130741](https://github.com/e-mission/e-mission-phone/pull/1072/commits/313074123bf6c9e5b0470ef210eece70ebd97c41)